bm_scrapy/__main__.py

```diff
@@ -28,15 +28,15 @@ def run_code(args, commands_module=None):
 
 
 def describe_project():
-    # bm-describe-project command
+    """bm-describe-project command."""
```

Comment:
Docstring needs to be a sentence like `Describe project.`. A simple and direct sentence in present simple.

Reply:
Done